### DISCUSSION TITLE
recommend video on the new pipe

### DISCUSSION THREAD

**Discussion by 223311** [TeamNewPipe/NewPipe#11422] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Is it possible to add a recommended videos feature to new pipe. This feature is highly beneficial, enhancing user experience by providing personalized video suggestions. We've observed that GoTube(v3.6) and YouTube revanced offers a similar feature where users can see recommended videos without logging into their account. Implementing this in new pipe would greatly improve accessibility and user engagement.

**Comment by ShareASmile** [comment]
It is Not Planned.
